### PR TITLE
Graceful LocalTrajectoryUploader

### PR DESCRIPTION
**Pull request by gaschler**
FIXES=#1370

### CODE REVIEW

File: cartographer/cloud/internal/local_trajectory_uploader.cc [hunk 1]
```diff
@@ -188,9 +196,14 @@ void LocalTrajectoryUploader::ProcessSendQueue() {
   while (!shutting_down_) {
     auto sensor_data = send_queue_.PopWithTimeout(kPopTimeout);
     if (sensor_data) {
+      bool ok = TranslateTrajectoryId(sensor_data->mutable_sensor_metadata());
+      if (!ok) {
```
Comment (pifon2a) [inline cartographer/cloud/internal/local_trajectory_uploader.cc]: `if(!TranslateTrajectoryId(sensor_data->mutable_sensor_metadata()))` ?

**Comment by pifon2a** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user pifon2a. Merge queue now has a length of 3.

**Comment by wally-the-cartographer** [comment]
Refusing to merge, since the combined checks state is failure

**Pull request merged**